frontend/src/services/session.service.ts

```diff
@@ -82,6 +83,30 @@ export default class SessionService {
         return SessionService.getFromCache<DisplayedLanguage>('language', false);
     }
 
+    public static saveArchives(archives: Archive[], who: string): void {
+        let json = archives.map(archiveToJson);
+        SessionService.saveInCache<ArchiveJson[]>('archives' + who, json);
+    }
+
+    public static getArchives(who: string): Archive[] | null {
+        let archives = SessionService.getFromCache<ArchiveJson[]>('archives' + who);
+        return archives?.map(archiveFromJson) ?? null;
+    }
+
+    /**
+     * This is for generating a random archive locally
+     * @param who 
+     */
+    public static getRandomArchive(who: string): Archive | null {
```

Comment:
What is the purpose of this function? Aren't we going to have the backend handle the archive randomization?

Reply:
I make it just in case we need it in the future. Right now it is not used anywhere

Reply:
ah ok, in that case, we probably should just get rid of it for now. We want to keep things as clean as possible, and I don't think we will ever need to use this since we aren't planning on sending the entire archive to the frontend. Just a single random video from the database via the API.